.circleci/config.yml

```diff
@@ -213,7 +213,7 @@ jobs:
             just simulate-council # simulate again to make sure the json is still valid
   just_simulate_sc_rehearsal_2_0122:
     docker:
-      - image: us-docker.pkg.dev/oplabs-tools-artifacts/images/ci-builder:latest
+      - image: us-docker.pkg.dev/oplabs-tools-artifacts/images/ci-builder:v0.35.0
```

Comment:
I was getting a command not found error when running `just` on the step below. 
This change just ensures we're using the same docker image tag as the above jobs, and it fixes the ci failure.

Reply:
Sgtm, but when should we bump to a more recent ci-builder tag, since v0.35.0 is no longer the latest

Reply:
Latest breaks the build, and shouldn't imo block this pr